public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -1134,6 +1134,11 @@ export class DashboardModel implements TimeModel {
       return false;
     }
 
+    // Disable 'Add annotation' popover when the device cannot hover, and must "click" to trigger tooltip
+    if (window.matchMedia('(hover: none)').matches) {
+      return false;
+    }
```

Comment:
I don't think DashboardModel is the right place to make this change.

This function contains business logic around whether the user has permissions to add annotations, and I don't think it's the right place to then put UI concerns. I'm thinking about other panels/UIs which don't share the problem this PR aims to solve would lose ability to add annotations.

If the problem lies with the hover interaction in TimeSeries, shouldn't that be where the change is made?